monkey/monkey_island/cc/services/config_schema.py

```diff
@@ -150,7 +150,7 @@
                         "BackdoorUser"
                     ],
                     "title": "Back door user",
-                    "attack_techniques": []
+                    "attack_techniques": ["T1136"]
```

Comment:
Would be extra nice if you added the same mapping to the "communicate as a new user" PBA.